### ISSUE TITLE
训练问题，ImageNet

### ISSUE THREAD

**Issue by angel707** [CompVis/latent-diffusion#189]
训练自动编码器，报一下错误，请问是什么原因呢？
IndexError: Caught IndexError in DataLoader worker process 0.
Original Traceback (most recent call last):

IndexError: list index out of range

**Comment by hbaghramyan** [comment]
Hey,

I am facing the same issue. Are you training on ImageNet?

**Comment by CharlieJCJ** [comment]
Facing the same issue. I'm also training on ImageNet

**Comment by deeptimhe** [comment]
It seems the indices in imagenet_train_hr_indices.p larger than the number of imagenet training images.

I replaced ImageNetSRTrain with ImageNetTrain.

**Comment by dreamlychina** [comment]
if i want to train my own datasets，should I train the autoencoder first using my own datasets??

**Comment by dreamlychina** [comment]
> 

大佬，你跑通了？求指导

**Comment by hbaghramyan** [comment]
> if i want to train my own datasets，should I train the autoencoder first using my own datasets??

nah, it is not a strict requirement although desired. You can also use one of the autoencoders (the checkpoints) on the repo. It would give you an idea of how the model performs. If it is good, then you may go and train your own autoencoder. But for the first check-up, I would go with a trained autoencoder.

**Comment by hellohahaw** [comment]
> > 
> 
> 大佬，你跑通了？求指导

大佬跑通了吗？能不能指导一下

**Comment by ALABAMABANANA** [comment]
> > 
> 
> 大佬，你跑通了？求指导

同学你好 请问最好跑通了吗？谢谢

**Comment by 77JULIE** [comment]
> > > 
> > 
> > 
> > 大佬，你跑通了？求指导
> 
> 大佬跑通了吗？能不能指导一下

你好打扰了，请问这个错误解决了吗，可以指导一下吗
